templates/drupal/0/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '2'
+services:
+  web:
+    image: drupal:${drupal_tag}
```

Comment:
@galal-hussein Thanks for the feedback. I can certainly make those changes however, creating custom catalogs, in my opinion, does not appear to be the best use of the versioning system for both drupal and mysql since the configurations could be many. The variable option allows a user to say I want to use drupal image version X with mysql Y to match the users development or production environment. Using the template will be unpractical.